lib/models/MatrixAction.js

```diff
@@ -45,7 +45,13 @@ MatrixAction.fromEvent = function(client, event) {
                 fileSize = " (" + Math.round(event.content.info.size / 1024) +
                     "KB)";
             }
-            text = client.mxcUrlToHttp(event.content.url) +
+
+            // By default assume that the media server = client homeserver
+            if (!mediaUrl) {
+                mediaUrl = client.getHomeserverUrl();
+            }
+
+            text = client.mxcUrlToAlternateHttp(mediaUrl, event.content.url) +
```

Comment:
So the `ContentRepo` class is currently being exported from the js-sdk @ https://github.com/matrix-org/matrix-js-sdk/blob/master/lib/matrix.js#L53 - I would much rather we exposed the `js-sdk` from `matrix-appservice-bridge` much like we do with `AppServiceRegistration` at https://github.com/matrix-org/matrix-appservice-bridge/blob/master/lib/exports.js#L29 than add this function to the client.